resources/main_test.go

```diff
@@ -0,0 +1,132 @@
+package resources
```

Comment:
Maybe we should put the whole test helper code into its own package? This would prevent cluttering up the `resources` package.